src/styles/blocks/stopwatch.scss

```
@@ -1,16 +1,16 @@
 .stopwatch {
+  background: center / cover no-repeat url(../images/clockface-12.jpg);
 
-  background: center no-repeat url(../images/clockface-12.jpg);
-  background-size: cover;
+  @include animationRotate(clockwise, -90deg, 270deg);
 
   &__second-arrow {
     position: absolute;
     left: 50%;
-    top: 49%;
+    top: calc(#{$stopwatch-size} / 2 - #{$arrow-height} / 2);

```

```suggestion
    top: calc((#{$stopwatch-size} - #{$arrow-height})/ 2);
```
you can simplify or use `50%` instead of `#{$stopwatch-size} / 2`